rfc/text/basic/bp_02_building_blocks.md

```diff
@@ -279,6 +279,7 @@ Following scenarios have to be considered protocol errors:
  - Receiving `REGISTERED` message, before session was established.
  - Receiving `UNREGISTERED` message, before session was established.
  - Receiving `INVOCATION` message, before session was established.
+ - Receiving message with non-sequential request ID, such as `SUBSCRIBE`, `UNSUBSCRIBE`, `PUBLISH`, `REGISTER`, `UNREGISTER`, `CALL` and `YIELD`.
```

Comment:
Well, here are listed only messages from client peers to the router. But I think this also applies to the reverse direction.

Reply:
Hm... Thinking... Are there any messages that can be initiated by router?... Seems that no. Only responses. So, sorry for the confusion.